src/main/java/net/sf/jabref/importer/fileformat/MedlineImporter.java

```diff
@@ -172,6 +183,33 @@ public ParserResult importDatabase(BufferedReader reader) throws IOException {
         return new ParserResult(bibItems);
     }
 
+    private int SkipReaderToFirstElement(BufferedReader reader) {
```

Comment:
XML parsers should do that work. Do not walk through XML by hand.

Please remove this method and try the solution presented at http://stackoverflow.com/a/22204172/873282

`xsr.next();` from XMLStreamReader seems to be your friend.